lib/rmt/scc.rb

```diff
@@ -52,6 +52,8 @@ def export(path)
 
     @logger.info('Exporting products')
     File.write(File.join(path, 'organizations_products.json'), scc_api_client.list_products.to_json)
+    # For SUMA, we also export the unscoped products.
+    File.write(File.join(path, 'organizations_products_unscoped.json'), scc_api_client.list_products_unscoped.to_json)
```

Comment:
For SUMA it'd be better to rename it to `organizations_products.json`, and rename the scoped one `organizations_products_scoped.json` and use that for offline RMT. (Otherwise SUMA changes are required)